opm/autodiff/BlackoilTransportModel.hpp

```diff
@@ -158,6 +158,13 @@ namespace Opm {
 
 
 
+        std::vector<V>
+        computeFluidInPlace(const ReservoirState& x,
+                            const WellState& xw,
+                            const std::vector<int>& fipnum)
+        {
+            return asImpl().computeFluidInPlace(x, xw, fipnum);
+        }
```

Comment:
>We should try to figure out why. Did you use a debugger to see what causes the segfault?

After debugging, I think the problem is that ``computeFluidInPlace()`` function calls some private or protected data members which can not be called in sequential solves,  the solution could be implement this function inside sequential solver instead. @atgeirr Do you have some suggestion?